subprojects/build-init/src/main/java/org/gradle/buildinit/tasks/InitBuild.java

```diff
@@ -67,6 +71,20 @@ public class InitBuild extends DefaultTask {
     @Internal
     private ProjectLayoutSetupRegistry projectLayoutRegistry;
 
+    /**
+     * Should we allow any existing files in the current directory to be overwritten?
+     *
+     * This property can be set via command-line option '--overwrite'. Defaults to true.
+     *
+     * @since TODO
```

Comment:
I am not sure on the version which should go here.

Reply:
8.0